src/backend/catalog/pipeline_query.c

```diff
@@ -342,33 +342,35 @@ GetCVNameFromMatRelName(RangeVar *name)
 }
 
 /*
- * GetQueryString
- *
- * Retrieves the query string of a registered continuous
- * view. If `select_only` is true, only the SELECT portion
- * of the query string is returned.
+ * GetContSelectStmt
  */
-char *
-GetQueryString(RangeVar *cvname)
+SelectStmt *
+GetContSelectStmt(RangeVar *rv)
 {
 	HeapTuple tuple;
 	bool isnull;
 	Datum tmp;
-	char *result = NULL;
+	char *sql;
+	Query *query;
+	SelectStmt *select;
```

Comment:
We use the same variable name in `cont_analyze.c` but we should probably stop using it since it's also a syscall...